src/wolfgame.py

```diff
@@ -767,7 +767,11 @@ def replace(cli, nick, chan, rest):
                 cli.notice(nick, msg)
             return
     else:
-        target, _ = complete_match(rest[0], var.list_players() + list(var.VENGEFUL_GHOSTS.keys()))
+        plyr_list = var.list_players + list(var.VENGEFUL_GHOSTS.keys())
+        lc_list   = [i.lower() for x in plyr_list]
```

Comment:
Use `pl` for player list and `pll` for lowercased player list for consistency with rest of lykos code. Also, please use IRC lowercasing instead of str.lower() -- I know existing code incorrectly uses str.lower() but that doesn't give an excuse for new code to. The IRC lowercasing function is in src/settings.py iirc (aka `var`).

Also, **test your code** before submitting a PR next time; it's pretty obvious that you didn't test this at all given the egregious error in it: **i**.lower() for **x**... A PR should only be submitted if whatever is changed has actually been verified to work and not blow up.